plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestParquetPageSkipping.java

```diff
@@ -146,4 +176,11 @@ private int assertSameResults(String query)
                 {"custkey", "smallint", new Object[][] {{4, 634, 640, 1493}}}
         };
     }
+
+    private Session noColumnIndexFiltering()
```

Comment:
`noParquet...`

pass initial  `session` as an argument, for more flexibility

Reply:
done